homeassistant/const.py

```diff
@@ -1163,6 +1164,11 @@ class UnitOfDataRate(StrEnum):
 
 SIGNAL_BOOTSTRAP_INTEGRATIONS = "bootstrap_integrations"
 
+# Date/Time formats
+FORMAT_DATE: Final = "%Y-%m-%d"
+FORMAT_TIME: Final = "%H:%M:%S"
+FORMAT_DATETIME: Final = f"{FORMAT_DATE} {FORMAT_TIME}"
+
```

Comment:
Partially unrelated to this PR, and no longer needed with the suggestions made in this review.